hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetOwnerRequest.java

```diff
@@ -65,12 +65,12 @@ public OMVolumeSetOwnerRequest(OMRequest omRequest) {
   public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
 
     long modificationTime = Time.now();
-    SetVolumePropertyRequest modifiedRequest = getOmRequest()
+    SetVolumePropertyRequest.Builder modifiedRequestBuilder = getOmRequest()
```

Comment:
The variable name is a little confused IMHO.
Do you mind that we change it to setPropertyRequestBuilder ?